sources-dist-stable.json

```diff
@@ -100,9 +100,9 @@
   "daswetter": {
     "meta": "https://raw.githubusercontent.com/rg-engineering/ioBroker.daswetter/master/io-package.json",
     "icon": "https://raw.githubusercontent.com/rg-engineering/ioBroker.daswetter/master/admin/daswettercom.png",
-    "version": "1.0.0",
+    "version": "1.0.1",
     "type": "weather",
-    "published": "2017-05-14T10:42:31.173Z"
+    "published": "2018-01-13T18:51:31.173Z"
```

Comment:
published must be set only by first time. 
It is time, when the adapter appears for the first time.